src/components/snapshot/ManagedSnapshotAccess.test.tsx

```diff
@@ -35,39 +47,46 @@ const initialState = {
     ],
     userRoles: ['steward', 'reader', 'discoverer', 'aggregate_data_reader'],
   },
+  datasets: datasetState,
+};
+
+const setup = (state) => {
```

Comment:
Optional: If you wanted, you could keep this to `stateOverrides` and then the mockstore could be: `{ ...initialState, ...stateOverrides }`?

Reply:
Discussed offline, because of nested properties this isn't that simple.